plugins/inputs/elasticsearch/elasticsearch.go

```diff
@@ -114,6 +114,13 @@ const sampleConfig = `
   ## Only gather cluster_stats from the master node. To work this require local = true
   cluster_stats_only_from_master = true
 
+  ## Set indices_stats to true when you want to obtain indices stats from the Master node.
+  indices_stats = false
+
+  ## Set shards_stats to true when you want to obtain shards stats from the Master node.
+  ## If set, then indices_stats is considered true as they are also provided with shard stats.
```

Comment:
The indices_level need to replace cluster_health_level also then?

Reply:
I don't think so, this is different data and I think one might want it at a different level.

Reply:
Hmm? But why do we have 'cluster' and 'indices' as option then?
Cause according to: https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-stats.html , there is only:

> In order to get back shard level stats, set the level parameter to shards.

No other levels ..

Reply:
I've only looked on ES6, but the call itself advertises these other settings: `level parameter must be one of [cluster] or [indices] or [shards] but was [howdy]`, from what I can tell they seem to work.  Indices is the same as leaving the value off, so this is just a way to combine the need for multiple booleans into a single option.

Reply:
Ah but that is https://www.elastic.co/guide/en/elasticsearch/reference/6.2/cluster-health.html
But the indices_stats come from https://www.elastic.co/guide/en/elasticsearch/reference/6.2/indices-stats.html
The only option there is "In order to get back shard level stats, set the level parameter to shards."

Make indices_shards = true/false? Or keep it indices_level = null/"shards" (if in the future additional options get added for example?)

Reply:
No, this is separate from `/_cluster/heath` and the output is different.  Specifically these paths:
```
/_all/_stats?level=shards
/_all/_stats?level=indices (appears to be the default /_all/_stats)
/_all/_stats?level=cluster
```

All of these work and give different information, though only "shards" is documented.  Let's do the string version of the option so that if we want to add some of the other options in the future we can.